code/sorting/src/topological_sort/readme.md

```diff
@@ -0,0 +1,31 @@
+# Topological Sort
+
+It is a simple algorithm that outputs linear ordering of vertices/nodes of DAG(Directed Acyclic Graph), where for each directed edge from node A to node B(a :arrow_right: B), node A appears before node B in ordering.
+
+## Condition:
+1. The graphs should be **directed**: otherwise for any two vertices a & b, 
+there would be two paths from a to b & vice-versa, and hence they cannot be ordered.
+
+2. The graphs should be **acyclic**: otherwise for any three vertices a,b,c on a cycle, there will not be any starting point, & hence, linear ordering can't be done.
+
+> Note: Via this algorithm, multiple orderings are possible.
+
+For example:
```

Comment:
A citation on the lines of: Image Credits: Simpson College Computer Science Faculty would be a good idea.